tracing-core/src/dispatcher.rs

```diff
@@ -345,6 +346,7 @@ where
 ///
 /// [dispatcher]: ../dispatcher/struct.Dispatch.html
 #[cfg(not(feature = "std"))]
+#[cfg_attr(docsrs, doc(cfg(feature = "std")))]
```

Comment:
this should be removed — note that the cfg_attr is `cfg(not(feature = "std"))`. the `get_default` fn will be present regardless, it just has a different implementation.